lib/matplotlib/mathtext.py

```diff
@@ -1595,7 +1590,7 @@ def _set_glue(self, x, sign, totals, error_type):
             self.glue_ratio = 0.
         if o == 0:
             if len(self.children):
-                warnings.warn(
+                cbook._warn_external(
```

Comment:
also logging?  a bit borderline but consistent with the rest of the mathtext module